weather/www/js/app.js

```diff
@@ -0,0 +1,147 @@
+
+    $scope.hour1 = time(1) + ", " + Math.round($scope.weather.hourly.data[1].temperature) + "°F and " +
+      $scope.weather.hourly.data[1].summary;
+    $scope.hour2 = time(2) + ", " + Math.round($scope.weather.hourly.data[2].temperature) + "°F and " +
+      $scope.weather.hourly.data[2].summary;
+    $scope.hour3 = time(3) + ", " + Math.round($scope.weather.hourly.data[3].temperature) + "°F and " +
+      $scope.weather.hourly.data[3].summary;
+    $scope.hour4 = time(4) + ", " + Math.round($scope.weather.hourly.data[4].temperature) + "°F and " +
+      $scope.weather.hourly.data[4].summary;
+    $scope.hour5 = time(5) + ", " + Math.round($scope.weather.hourly.data[5].temperature) + "°F and " +
+      $scope.weather.hourly.data[5].summary;
+    $scope.hour6 = time(6) + ", " + Math.round($scope.weather.hourly.data[6].temperature) + "°F and " +
+      $scope.weather.hourly.data[6].summary;
+    $scope.hour7 = time(7) + ", " + Math.round($scope.weather.hourly.data[1].temperature) + "°F and " +
+      $scope.weather.hourly.data[7].summary;
+    $scope.hour8 = time(8) + ", " + Math.round($scope.weather.hourly.data[2].temperature) + "°F and " +
+      $scope.weather.hourly.data[8].summary;
+    $scope.hour9 = time(9) + ", " + Math.round($scope.weather.hourly.data[3].temperature) + "°F and " +
+      $scope.weather.hourly.data[9].summary;
+    $scope.hour10 = time(10) + ", " + Math.round($scope.weather.hourly.data[4].temperature) + "°F and " +
+      $scope.weather.hourly.data[10].summary;
+    $scope.hour11 = time(11) + ", " + Math.round($scope.weather.hourly.data[5].temperature) + "°F and " +
+      $scope.weather.hourly.data[11].summary;
+    $scope.hour12 = time(12) + ", " + Math.round($scope.weather.hourly.data[6].temperature) + "°F and " +
+      $scope.weather.hourly.data[12].summary;
+
+    $scope.day1 = dayOfTheWeek(1) + ", " + Math.round($scope.weather.daily.data[1].temperatureMax) + "°F and " +
+      $scope.weather.daily.data[1].summary;
+    $scope.day2 = dayOfTheWeek(2) + ", " + Math.round($scope.weather.daily.data[2].temperatureMax) + "°F and " +
+      $scope.weather.daily.data[2].summary;
+    $scope.day3 = dayOfTheWeek(3) + ", " + Math.round($scope.weather.daily.data[3].temperatureMax) + "°F and " +
+      $scope.weather.daily.data[3].summary;
+    $scope.day4 = dayOfTheWeek(4) + ", " + Math.round($scope.weather.daily.data[4].temperatureMax) + "°F and " +
+      $scope.weather.daily.data[4].summary;
+    $scope.day5 = dayOfTheWeek(5) + ", " + Math.round($scope.weather.daily.data[5].temperatureMax) + "°F and " +
+      $scope.weather.daily.data[5].summary;
+    $scope.day6 = dayOfTheWeek(6) + ", " + Math.round($scope.weather.daily.data[6].temperatureMax) + "°F and " +
+      $scope.weather.daily.data[6].summary;
+    $scope.day7 = dayOfTheWeek(7) + ", " + Math.round($scope.weather.daily.data[7].temperatureMax) + "°F and " +
+      $scope.weather.daily.data[7].summary;
+
```

Comment:
So i tried using a for loop, however it says the variable becomes undefined for both for loops, I will look for a solution at a later date